.github/workflows/nix-action.yml

```diff
@@ -37,16 +35,16 @@ jobs:
           else
             echo "tested_commit=${{ github.sha }}" >> $GITHUB_ENV
           fi
-      - uses: cachix/install-nix-action@v16
+      - uses: cachix/install-nix-action@v20
         with:
           nix_path: nixpkgs=channel:nixpkgs-unstable
-      - uses: cachix/cachix-action@v10
+      - uses: cachix/cachix-action@v12
         with:
           name: coq-community
           authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
           extraPullNames: coq, math-comp
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           ref: ${{ env.tested_commit }}
       - run: >
-          nix-build https://coq.inria.fr/nix/toolbox --argstr job reglang --arg override '{ ${{ matrix.overrides }}; reglang = builtins.filterSource (path: _: baseNameOf path != ".git") ./.; }'
+          nix-build https://coq.inria.fr/nix/toolbox --argstr job reglang --arg override '{ ${{ matrix.overrides }}; mathcomp = "2.0.0"; reglang = builtins.filterSource (path: _: baseNameOf path != ".git") ./.; }'
```

Comment:
This is a deviation from the template-generated file that will have to be maintained until the reglang definition from nixpkgs used in the toolbox is changed to depend on mathcomp 2.0.0. An alternative would be to replace this basic `nix-action` by a full Coq Nix Toolbox installation.

Reply:
Ok, let me introduce the toolbox in a separate PR.